daffodil-core/src/main/scala/org/apache/daffodil/dsom/IIBase.scala

```diff
@@ -133,8 +133,8 @@ abstract class IIBase( final override val xml: Node, xsdArg: XMLSchemaDocument,
    * schema file, recursively any of its includes/imports,
    * and that all the resulting are validated by the validating loader.
    */
-  requiredEvaluations(iiSchemaFile)
-  requiredEvaluations(iiSchemaFile.iiXMLSchemaDocument)
+  requiredEvaluations(iiSchemaFileMaybe)
```

Comment:
Please add code comment as to why this does the "right thing" - avoids redundant loading.